docs/monitoring/10-01-prometheus-istio-server-oom.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Prometheus Istio Server restarting or in crashback loop due to OOM
+type: Troubleshooting
+---
+
+Prometheus Istio Server scrapes metricd from all the envoy side car. It can restart or have crashbackloop due to insufficient memory. This issue can be tackled in following ways:
```

Comment:
```suggestion
Prometheus Istio Server scrapes metrics from all envoy side cars. It can restart or have a crashback loop due to insufficient memory (OOM). This issue can be tackled by following these steps:
```